Samples/NetworkMediaStreamer/RTPMediaStreamer/build/RTPMediaStreamer.vcxproj

```diff
@@ -0,0 +1,201 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Label="ProjectConfigurations">
+    <ProjectConfiguration Include="Debug|Win32">
+      <Configuration>Debug</Configuration>
+      <Platform>Win32</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|Win32">
+      <Configuration>Release</Configuration>
+      <Platform>Win32</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Debug|x64">
+      <Configuration>Debug</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|x64">
+      <Configuration>Release</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+  </ItemGroup>
+  <ItemGroup>
+    <ClInclude Include="..\..\Common\inc\RTPMediaStreamer.h" />
+    <ClInclude Include="..\inc\pch.h" />
+    <ClInclude Include="..\inc\RTPStreamSink.h" />
+  </ItemGroup>
+  <ItemGroup>
+    <ClCompile Include="..\src\RTPMediaSink.cpp" />
+    <ClCompile Include="..\src\RTPStreamSink.cpp" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\NetworkMediaStreamerBase\build\NetworkMediaStreamer.vcxproj">
+      <Project>{58e4848e-b347-43fb-94c1-c5c85abfd6b0}</Project>
+    </ProjectReference>
+  </ItemGroup>
+  <PropertyGroup Label="Globals">
+    <VCProjectVersion>16.0</VCProjectVersion>
+    <ProjectGuid>{763269FF-37EB-4A0A-B152-0924CFA81FFF}</ProjectGuid>
+    <Keyword>Win32Proj</Keyword>
+    <RootNamespace>VideoStreamer</RootNamespace>
+    <WindowsTargetPlatformVersion>10.0.18362.0</WindowsTargetPlatformVersion>

```

Comment:
>18362 [](start = 39, length = 5)

Top level readme.md says 17763. Either update it or in local readme.md mention the 18362 (19H1), or make it work with 17763 (RS5).

These samples require Visual Studio 2017 Update 4 or higher and the Windows Software Development Kit (SDK) version 17763 for Windows 10.

Reply:
The code works with 17763, but i cannot test the build with VS 2017.. as its too old... should i just modify top level readme to use VS 2019? 

---
In reply to: [491066730](https://github.com/microsoft/Windows-Camera/pull/13#discussion_r491066730) [](ancestors = 491066730)